app/src/main/java/com/nearbynest/app/ui/home/HomeViewModel.kt

```diff
@@ -0,0 +1,25 @@
+/**
+ * Copyright COPYRIGHT
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.nearbynest.app.ui.home
+
+import com.nearbynest.app.ui.base.BaseViewModel
+import timber.log.Timber
+
+class HomeViewModel : BaseViewModel() {
+    init {
+        Timber.e("Initialized")
```

Comment:
Why `e` ?

Reply:
I understood `e` is used for errors.
Replaced it with 'd'.